### PR TITLE
workflows: add docs for Revive workflow

### PR DESCRIPTION
**Pull request by KexyBiscuit**
- Create a comment contains link to the run after invoked.

### CODE REVIEW

File: .github/workflows/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## Syntax
+
+```
+/build <Package> [<Package> ...]
+```
+
+## Description
+
+The `/build` command builds packages on GitHub Actions. It starts clean, up-to-date instances of buildkit container, runs `acbs-build` to build packages.
```
Comment (MingcongBai) [inline .github/workflows/README.md]: via GitHub Actions. *

File: .github/workflows/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+When replying the command under an issue, packages are built from `stable` branch. When replying the command under a pull request, packages are built from the pull request.
```
Comment (MingcongBai) [inline .github/workflows/README.md]: When replying the command under an issue, packages are built from `stable` branch. When replying the command under a pull request, packages are built from the respective origin branch of the pull request.

File: .github/workflows/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+## Examples
+
+### Example 1: Build one package
```
Comment (MingcongBai) [inline .github/workflows/README.md]: Building *, same for below.

File: .github/workflows/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+### Example 3: Build a group of packages
+
+```
+/build groups/dotnet
+```
+
+## Notes
+
+- Currently only GitHub-hosted runners are supported.
```
Comment (MingcongBai) [inline .github/workflows/README.md]: Currently, this command only supports GitHub-hosted runners.

File: .github/workflows/README.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  - Standard_DS2_v2 virtual machines in Microsoft Azure isn't big enough to build every package in AOSC OS, nor does it provide architectures other than Intel 64.
```
Comment (MingcongBai) [inline .github/workflows/README.md]: Microsoft Azure's Standard_DS2_v2 virtual machines only support the x86_64 (`amd64`) architecture, and may not have sufficient storage to build some packages.

File: .github/workflows/README.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+## Related Links
```
Comment (MingcongBai) [inline .github/workflows/README.md]: Further Reading

File: README.md [hunk 7]
```diff
@@ -24,6 +24,11 @@ it is possible to build a series of packages with a single command. This is
 achieved by the "groups" defined in the `groups/` directory, with files defining
 a list of packages to build.
 
+Automation
+----------
+
+Under GitHub issues and pull requests, one can directly invoke package building on GitHub Actions, see [reference of workflows](.github/workflows/README.md) for details.
```
Comment (MingcongBai) [inline README.md]: You may initiate package build tests via GitHub Actions.

_One does not simply use "one."_

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: .github/workflows/README.md (hunk 1 above)
Comment (KexyBiscuit) [reply .github/workflows/README.md]:
```suggestion
The `/build` command builds packages via GitHub Actions. It starts clean, up-to-date instances of buildkit container, runs `acbs-build` to build packages.
```

File: .github/workflows/README.md (hunk 2 above)
Comment (KexyBiscuit) [reply .github/workflows/README.md]:
```suggestion
When replying the command under an issue, packages are built from `stable` branch. When replying the command under a pull request, packages are built from the source branch of the pull request.
```

File: .github/workflows/README.md (hunk 3 above)
Comment (KexyBiscuit) [reply .github/workflows/README.md]:
```suggestion
### Example 1: Building One Package
```

File: .github/workflows/README.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+## Examples
+
+### Example 1: Build one package
+
+```
+/build aosc-aaa
+```
+
+### Example 2: Build multiple packages
```
Comment (KexyBiscuit) [inline .github/workflows/README.md]:
```suggestion
### Example 2: Building Multiple Packages
```

File: .github/workflows/README.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+```
+/build openssl openssl+32
+```
+
+### Example 3: Build a group of packages
```
Comment (KexyBiscuit) [inline .github/workflows/README.md]:
```suggestion
### Example 3: Building A Group Of Packages
```

File: .github/workflows/README.md (hunk 4 above)
Comment (KexyBiscuit) [reply .github/workflows/README.md]:
```suggestion
- Currently, this command only supports GitHub-hosted runners.
```

File: .github/workflows/README.md (hunk 5 above)
Comment (KexyBiscuit) [reply .github/workflows/README.md]:
```suggestion
  - Microsoft Azure's Standard_DS2_v2 virtual machines only support x86-64 (`amd64`) architecture and may not have sufficient performance or storage to build some packages.
```

File: .github/workflows/README.md (hunk 6 above)
Comment (KexyBiscuit) [reply .github/workflows/README.md]:
```suggestion
## Further Reading
```

File: README.md (hunk 7 above)
Comment (KexyBiscuit) [reply README.md]:
```suggestion
Under GitHub issues and pull requests, you may initiate package build tests via GitHub Actions, see [reference of workflows](.github/workflows/README.md) for details.
```

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**